Marlin/example_configurations/Cartesio/Configuration_adv.h

```diff
@@ -224,6 +224,16 @@
 #define EXTRUDER_AUTO_FAN_TEMPERATURE 35
 #define EXTRUDER_AUTO_FAN_SPEED   255  // == full speed
 
+// Define a automatic fan pin to cooldown heatbed
+// 
+// When the delta temperature between current and target bed temperature is big, 
+// heatbed cooldown fan will automaticly blow to accelerate the cooldown proccess.
+// Useful when you have a PEI build surface.
+#define HEATBED_AUTO_FAN_PIN -1
```

Comment:
As mentioned, for completeness, you can define the pin that will be used as the default `HEATBED_AUTO_FAN_PIN` in each `pins_SOMEBOARD.h` file. For `HEATBED_AUTO_FAN_PIN` this is easy to figure out. The pin will be same as `FAN1_PIN` or `FAN2_PIN`. So in `pins_RAMBO.h`, the block would be:
```cpp
#ifndef HEATBED_AUTO_FAN_PIN 
  #define HEATBED_AUTO_FAN_PIN 2
#endif
```
…which corresponds to its `FAN2_PIN`. (Just using `FAN2_PIN` should also work in most cases, but maybe not RAMPS-based.)

The `pins_MEGACONTROLLER.h` file has `CONTROLLERFAN_PIN` defined as the same as `FAN2_PIN`, and that particular conflict isn't handled in `Conditionals_post.h`, so that (line 451) would need to be tweaked.